synth.py

```diff
@@ -46,8 +46,11 @@
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(cov_level=75)
-s.move(templated_files)
+templated_files = common.py_library(
+    samples=False,  # set to True only if there are samples
+    microgenerator=True,
```

Comment:
You can add `cov_level=99` here and the noxfile will use that value.

Reply:
done, it works, thanks!